pages.ru/android/am.md

```diff
@@ -0,0 +1,20 @@
+# am
+
+> Менеджер активностей Android.
+> Дополнительная информация: <https://developer.android.com/studio/command-line/adb#am>.
```

Comment:
We use this standard from https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py
```suggestion
> Больше информации: <https://developer.android.com/studio/command-line/adb#am>.
```